### PR TITLE
[#72] Make undefined producer side tests pending

### PR DESCRIPTION
**Pull request by kayvonghaffari**
[#72]

### CODE REVIEW

**Comment by kayvonghaffari** [comment]
screen shot 2015-12-16 at 10 42 23 pm

File: lib/rester/rspec.rb [hunk 1]
```diff
@@ -37,67 +37,71 @@
...
-
-    ##
-    # Raw response from the service.
-    # [HTTP CODE, JSON String]
-    ex.example_group.let(:raw_service_response) {
-      @rester_adapter.request(verb.downcase.to_sym, path, spec['request'])
-    }
+    unless ex.pending?
+      # Gather the request args from the spec descriptions
+      #
```
Comment (roberthoner) [inline lib/rester/rspec.rb]: how about just `return if ex.pending?` since there's so much else in this block.  No need for it to all be tabbed over.

Comment (kayvonghaffari) [inline lib/rester/rspec.rb]: it's an unexpected return since there's no enclosing method for the block

File: lib/rester/rspec.rb [hunk 2]
```diff
@@ -116,7 +120,9 @@ def _validate_test_coverage(ex)
 
         missing_contexts.each { |missing_context, _|
           context_group = _find_or_create_child(verb_group, missing_context)
-          context_group.it { is_expected.to include_stub_response }
+          context_group.pending('should be implemented') {
+            fail 'not implemented'
+          }
```
Comment (roberthoner) [inline lib/rester/rspec.rb]: How about just `context_group.pending`

image

File: lib/rester/rspec.rb (hunk 1 above)
Comment (roberthoner) [inline lib/rester/rspec.rb]: Hmm.. okay, how about wrapping the contents of the `unless` block in a helper?

```ruby
_setup_example(ex) unless ex.pending?
```

**Comment by kayvonghaffari** [comment]
Cool, made the changes. The error looks like this now (much better!):

screen shot 2015-12-17 at 1 19 36 pm

**Comment by roberthoner** [comment]
Code review looks good.

**Pull request merged**